_articles/maintaining-balance-for-open-source-maintainers.md

```diff
@@ -0,0 +1,220 @@
+
+<aside markdown="1" class="pquote">
+  [I would like to have] more financial support, so that I can focus on the open source work without burning through my savings and knowing I'll have to do a lot of contracting to make up for it later.
+  <p markdown="1" class="pquote-credit">
+— open source maintainer
+  </p>
+</aside>
+
+* **Conflicting demands:**  Open source is full of groups with different motivations, which can be difficult to navigate. If you're paid to do open source, your employer's interests can sometimes be at odds with the community.
+
+<aside markdown="1" class="pquote">
+  With paid open source, conflict between employer's focus and what's best for the community
+  <p markdown="1" class="pquote-credit">
+— open source maintainer
+  </p>
+</aside>
+
+### Watch out for signs of burnout
+
+Can you keep up your pace for 10 weeks? 10 months? 10 years?
+
+There are tools like the [Burnout Checklist](https://governingopen.com/resources/signs-of-burnout-checklist.html) from [@shaunagm](https://github.com/shaunagm) and Mozilla's [personal ecology self-assessment kit](https://docs.google.com/document/d/1duOYQ6EbcDTH_CK6ux3BGRiVYptGTUMOtndZbbwulOY/edit#heading=h.mn38481ischw) that can help you reflect on your current pace and see if there are any adjustments you can make. Some maintainers also use wearable technology to track metrics like sleep quality and heart rate variability (both linked to stress).
+
+<aside markdown="1" class="pquote">
+ I'm a big believer in good wearables. With the science behind it, you can understand how you could have done better and how to get to an optimal state of what you want to do.
+  <p markdown="1" class="pquote-credit">
+— open source maintainer
+  </p>
+</aside>
+
+### What would you need to continue sustaining yourself and your community?
+
+This will look different for each maintainer, and will change depending on your phase of life and other external factors. But here are a few themes we heard:
+
+* **Lean on the community:** Delegation and finding contributors can alleviate the workload. Having multiple points of contact for a project can help you take a break without worrying. Connect with other maintainers and the wider community–in groups like the [Maintainer Community](http://maintainers.github.com/). This can be a great resource for peer support and learning. 
+
+  You can also look for ways to engage with the user community, so you can regularly hear feedback and understand the impact of your open source work.
+
+* **Explore funding:** Whether you're looking for some pizza money, or trying to go full time open source, there are many resources to help! As a first step, consider turning on [GitHub Sponsors](https://github.com/sponsors) to allow others to sponsor your open source work. If you're thinking about making the jump to full-time, apply for the next round of [GitHub Accelerator](http://accelerator.github.com/).
```

Comment:
[nitpick] Hmmm..., I don't think GH Sponsors necessarily has to be a first step. It could be an option that maintainers can use. Reframing it as something to consider and mentioning other programs would put less pressure on maintainers to only use GH Sponsors. 
```suggestion
* **Explore funding:** Whether you're looking for some pizza money, or trying to go full time open source, there are many resources to help! Consider turning to [GitHub Sponsors](https://github.com/sponsors), [Stackaid](https://www.stackaid.us/), or [Open Collective](https://opencollective.com/) to allow others to sponsor your open source work. 
If you're thinking about making the jump to full-time, apply for the next round of [GitHub Accelerator](http://accelerator.github.com/). 
```